apps/examples/e2e/tests/test-shapes.spec.ts

```diff
@@ -111,9 +106,9 @@ test.describe('Shape Tools', () => {
 		for (const { tool, shape } of clickableShapeCreators) {
 			// Find and click the button
 			if (!(await page.getByTestId(`tools.${tool}`).isVisible())) {
-				await page.getByTestId('tools.more').click()
+				await page.getByTestId('tools.more-button').click()
```

Comment:
Maybe use `toolbar.moreToolsButton` here as well (there's a few more further down the file).